frontend/src/components/SidebarLayout/index.js

```diff
@@ -5,24 +5,31 @@ import {
   InstrumentsItem,
   Hero
 } from "@code4ro/taskforce-fe-components";
-import { instrumentsItems } from "./const";
+import { browserExtensions, instrumentsItems } from "./const";
 
 const SidebarLayout = ({ children }) => {
+  // Browser detection based on this answer:
+  // https://stackoverflow.com/a/9851769/5723188
+  const isChrome =
```

Comment:
FYI: this doesn't seem to work on Chrome mobile.
![Screenshot_20200628-003042](https://user-images.githubusercontent.com/11520795/85932694-f324aa80-b8d6-11ea-837e-532fe6d4ff36.jpg)
But that should not matter as Chrome mobile doesn't support extensions yet.

The Firefox detection works on mobile, which matter the most

Reply:
Yeah, I think the mechanism for detecting Chrome relies on detecting Chrome _extension support_, so if mobile doesn't have it, it won't show up.